internal/pkg/deploy/deploy.go

```diff
@@ -160,7 +160,7 @@ func (s *Source) Owner() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return oAndR.repo, nil
+	return oAndR.owner, nil
```

Comment:
Is there a test that'd have caught this?

Reply:
yep there is an unit test which asserts the expected template output. I didn't check carefully enough when replacing the expected output with the incorrectly-generated output.

Reply:
yea there's an unit test that compares the generated template against an expected template. I didn't check the generated template carefully enough last time and just copied it over the expected template.